### PR TITLE
fix(TBD-11112): InvalidClassException when I use tFileOutputDelimited running on CDH611

### PR DESCRIPTION
**Pull request by sponomarova**
… running on CDH611

**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/template/cdhx/cdh6xTemplate.json [hunk 1]
```diff
@@ -629,7 +636,23 @@
 			"jarName": "hive-exec-shade-2.1.1-cdh6.1.1-talend-1.1.jar",
 			"mvnUri": "mvn:org.talend.bigdata.libs/hive-exec-shade-2.1.1-cdh6.1.1-talend/1.1",
 			"useStudioRepository": "true",
-			"description": ""
+			"description": "",
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/resources/template/cdhx/cdh6xTemplate.json:639]: Is this intended ? I thought it causes issue to have exclusion with standard type

Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/template/cdhx/cdh6xTemplate.json:639]: You're right Alix, I think template will be unreadable. Have you tested the fix @sponomarova ?

Comment (CarlosTalend) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/template/cdhx/cdh6xTemplate.json:639]: Cf. https://talend.slack.com/archives/G9FMF7HGR/p1597732845382900

Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/template/cdhx/cdh6xTemplate.json]: Changed.

**Pull request merged**